examples/finetuning/dpo_tic_tac_toe/README.md

```
@@ -0,0 +1,724 @@
+# OpenAI-compatible API key for inference
+export OPENAI_API_KEY="unused-default-key"
+
+# NeMo Customizer service endpoints
+export CUSTOMIZER_HOST="https://your-nmp-host"
+export DATASTORE_HOST="https://your-datastore-host"
+export CUSTOMIZER_NIM_URL="https://your-nim-deployment-host"
+```
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)